akka-stream/src/main/scala/akka/stream/javadsl/Framing.scala

```diff
@@ -94,6 +94,33 @@ object Framing {
     scaladsl.Framing.lengthField(fieldLength, fieldOffset, maximumFrameLength, byteOrder).asJava
 
   /**
+   * Creates a Flow that decodes an incoming stream of unstructured byte chunks into a stream of frames, assuming that
+   * incoming frames have a field that encodes their length.
+   *
+   * If the input stream finishes before the last frame has been fully decoded this Flow will fail the stream reporting
+   * a truncated frame.
```

Comment:
I guess it lacks comma`... frame has been fully decoded _,_ this Flow will fail the ...`.
What do you mean about strangeness?
This is just copy from existing method.